cdc/api/owner/owner.go

```diff
@@ -227,18 +227,13 @@ func (h *ownerAPI) handleChangefeedQuery(w http.ResponseWriter, req *http.Reques
 	}
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
-	etcdClient, err := h.capture.GetEtcdClient()
-	if err != nil {
-		api.WriteError(w, http.StatusBadRequest, err)
-		return
-	}
-	cfInfo, err := etcdClient.GetChangeFeedInfo(ctx, changefeedID)
+	cfInfo, err := h.capture.GetEtcdClient().GetChangeFeedInfo(ctx, changefeedID)
```

Comment:
Is it not safe to cache etcdclient ?

Reply:
Yes, etcd client is initialized before creating `Capture`, and will not be closed before exiting.